types/skatejs/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "interface-over-type-literal": false
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`,
if necessary, you can disable some rules by adding `// tslint: disable-next-line:interface-over-type-literal` in source code.

Reply:
Ok fair enough